eth/stagedsync/stage_snapshots.go

```diff
@@ -418,6 +426,18 @@ func FillDBFromSnapshots(logPrefix string, ctx context.Context, tx kv.RwTx, dirs
 	return nil
 }
 
+func computeBlocksToPrune(cfg SnapshotsCfg) uint {
```

Comment:
We have method `cfg.prune.History.PruneTo(currentBlock)` - whats the difference with this method? can use existing?

Reply:
modified in a way that makes it return how many blocks to keep.